powerauth-push-server/src/main/java/io/getlime/push/repository/model/PushDeviceEntity.java

```diff
@@ -56,8 +56,8 @@
     @Column(name = "push_token", nullable = false)
     private String pushToken;
 
-    @Column(name = "last_registered", nullable = false)
-    private Date lastRegistered;
+    @Column(name = "timestamp_created", nullable = false)
```

Comment:
Same as above, the timestamp name must represent actual meaning of the timestamp.